main/src/ca/uwaterloo/flix/language/phase/Weeder.scala

```diff
@@ -526,8 +526,8 @@ object Weeder {
           @@(elms map visit) map {
             case es =>
               val empty = mkApply("Set/empty", Nil, sp1, sp2)
-              es.foldLeft(empty) {
-                case (acc, elm) => mkApply("Set/insert", List(elm, acc), sp1, sp2)
+              es.foldRight(empty) {
```

Comment:
Now #{1, 2, 3} translates to Set(List(1, 2, 3)) as opposed to Set(List(3, 2, 1)).